base/client.jl

```diff
@@ -125,14 +125,26 @@ function repl_cmd(cmd, out)
     nothing
 end
 
-display_error(er) = display_error(er, [])
-function display_error(er, bt)
-    print_with_color(Base.error_color(), STDERR, "ERROR: "; bold = true)
-    with_output_color(Base.error_color(), STDERR) do io
-        showerror(io, er, bt)
+function display_error(io::IO, er, bt)
+    if !isempty(bt)
+        st = stacktrace(bt)
+        if !isempty(st)
+            io = redirect(io, log_error_to, st[1])
+        end
+    end
+    print_with_color(Base.error_color(), io, "ERROR: "; bold = true)
+    # remove REPL-related frames from interactive printing
+    eval_ind = findlast(addr->Base.REPL.ip_matches_func(addr, :eval), bt)
+    if eval_ind != 0
+        bt = bt[1:eval_ind-1]
+    end
+    with_output_color(Base.error_color(), io) do io
```

Comment:
i'm confused.  can you be more precise in describing what you want changed?

julia v0.5.0 had no calls to `with_output_color` until [you](https://github.com/JuliaLang/julia/commit/2fc856d739d61656190956f7e63bce36375cd93e) added it to both `REPL.jl` and `client.jl`.  at some later point, it was deleted from `REPL.jl` but not `client.jl`.  are you saying i should delete your change from the latter as well?

it still seems to me that if the goal is that the output should be the same, then there should just be a single function, and not two.